lib/telemetry/helper.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Telemetry.Helper do
+  @moduledoc """
+  Helper functions for seeing all telemetry events.
+
+  Only for use in development.
+  """
+
+  require Logger
```

Comment:
Logger is required here, but I don't see it being used.  Should it be used over IO.inspect